addons/l10n_tw_edi_ecpay_website_sale/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "Taiwan - E-invoicing Ecommerce",
+    "version": "1.0",
```

Comment:
```suggestion
```
Default value for versions in manifests, can be dropped to keep the manifest as light as possible ;)